test/config_cmake.py

```diff
@@ -89,6 +89,9 @@ def main():
     add_arg(group, '--cuda-flags', 'CUDAFLAGS', 'CMAKE_CUDA_FLAGS', None,
             'STRING', 'CUDA compiler flags')
 
+    add_arg(group, '--cuda-std', 'CMAKE_CUDA_STANDARD', 'CMAKE_CUDA_STANDARD',
+            '11', 'STRING', 'CUDA standard')
```

Comment:
It's been a number of hours since I started reviewing this PR, but didn't this previously specify that CUDA support would henceforth require at least C++14?

Reply:
Yep, this needs to be updated.